aten/src/ATen/Dispatch.h

```diff
@@ -215,6 +215,23 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                       \
   }()
 
+#define AT_DISPATCH_FLOATING_AND_C10_COMPLEX_TYPES(TYPE, NAME, ...)         \
```

Comment:
Could you rename it to `_AT_DISPATCH_FLOATING_AND_C10_COMPLEX_TYPES`? After discussion with @gchanan , we think it is better to have a name starting with `_` to indicate that it is a temporary macro and will be removed eventually. There is nothing special you need to be careful with `c10::complex`, for most cases, it will be just like you are using `std::complex`.

Reply:
Thanks for you advices,! I'll fix it in the next commit.